bin/spark.js

```diff
@@ -1,8 +1,14 @@
 import http from 'node:http'
 import { once } from 'node:events'
-import { handle } from '../index.js'
+import { createHandler } from '../index.js'
+import pg from 'pg'
 
-const server = http.createServer(handle)
-server.listen(8080)
+const { PORT = 8080, DATABASE_URL } = process.env
+
+const client = new pg.Client({ connectionString: DATABASE_URL })
+await client.connect()
```

Comment:
I think we should use a connection pool, otherwise we cannot handle multiple incoming requests concurrently.

See https://node-postgres.com/apis/pool

Reply:
good call! https://github.com/filecoin-station/spark-api/pull/9